src/ROUTES.js

```diff
@@ -21,4 +21,10 @@ export default {
     DETAILS: 'details',
     DETAILS_WITH_LOGIN: 'details/:login',
     getDetailsRoute: login => `details/${login}`,
+    PARTICIPANTS: 'participants',
+    PARTICIPANTS_WITH_REPORT: 'participants/:reportID',
+    getParticipantsRoute: reportID => `participants/${reportID}`,
+    REPORT_ID: '/:reportID',
+    PARTICIPANTS_PARTICIPANT_DETAIL: '/participants/:reportID/:login',
+    getParticipantsPartipantDetail: (reportID, login) => `/participants/${reportID}/${login}`,
```

Comment:
"participants participant detail" could be more descriptive... e.g. maybe should be `REPORT_PARTICIPANTS` and `REPORT_PARTICIPANT`.

As for the routes, is there any reason we can't make these routes...

`/r/:reportID/participants`

`/r/:reportID/participants/:login`

?

Reply:
Yup we can do this, thanks for the suggestion.